.circleci/DSCSDSI_outputs.txt

```diff
@@ -26,13 +26,13 @@ qsiprep/sub-tester/dwi/sub-tester_acq-HASC55AP_confounds.tsv
 qsiprep/sub-tester/dwi/sub-tester_acq-HASC55AP_desc-ImageQC_dwi.csv
 qsiprep/sub-tester/dwi/sub-tester_acq-HASC55AP_desc-SliceQC_dwi.json
 qsiprep/sub-tester/dwi/sub-tester_acq-HASC55AP_dwiqc.json
```

Comment:
Do you think we should add space- tags to these too? Some of the QC metrics are calculated on the ACPC-resampled data

Reply:
I think it's reasonable to include it. I'll update accordingly.